[BACK]/app/controllers/User.js

```diff
@@ -32,11 +69,16 @@ exports.register = async function (req, res) {
             ))
         );
 
+        newUsers = newUsers.filter((user) =>
+            typeof user.email === 'string' &&
+            typeof user.teamId === 'number'
+        );
+
         if (newUsers.length) {
             newUsers.forEach(async newUser => {
                 const token = require('crypto').randomBytes(10).toString('hex');
                 console.log("token : ", token);
-                if (user.RoleId == 2) {
+                if (user.RoleId === 2 || true) {
```

Comment:
Why "true" ??

Reply:
My bad, I was testing something and forget to remove `true`